README.md

```diff
@@ -1,13 +1,19 @@
-# Project Name
+# Guess Who - Technigo Bootcamp Week 5
+My first JavaScript project on my own in week 5 of Technigo Bootcamp.
+The assigment was to create functions to the game. Sending messages/questions to the user, filter the characters out and reload the page. etc.. 
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I had problems to figure out quite a lot during the project. Turns out JavaScript is a bit tricky to understand. 
+
+I was lucky to have a team that helped out and supported me. I googled a lot, rewached the info videos/lectures from school and asked a lot of questions in the team. 
+
+During one point I could actually help explaining one part of code to a team member, an dthat helped me to even understand it a bit more by hearing my self explain it.
+
+If I had more time I would like to do more tutorials and catch up on the basics in JavaScript. I wish to work on really understand it in order to move forward. 
+With this project I was happy to meet the requierments but in the future I would love to add more features and fun functions to my code. 
 
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://app.netlify.com/sites/condescending-ramanujan-50f929/overview
```

Comment:
This Netlify link doesn't work. I would request that you replace this with a link that works 😊

Reply:
You need to change it to https://condescending-ramanujan-50f929.netlify.app/